experimental/devices/mfrc522/commands/low_level.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package commands
```

Comment:
What about have the package name imply that it's low level, then drop the "LowLevel" from the package's symbols? Then you can use New, Dev, etc. I'm not particularly fond of 'commands'.

Reply:
What about "raw"? And make raw.Dev to implement Resource (with Halt, as you've mentioned, calling irqPin.Halt).

Reply:
I'd prefer "mfrc522raw" or "mfrc522low". This is because when a symbol is referred to in client code, mfrc522raw.Dev makes more sense that raw.Dev, which is not informative.